### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by AndreyKozhevnikov**

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -1,24 +1,26 @@
...
-<!-- default file list -->
-*Files to look at*:
 
-* [CriteriaController.cs](CS/EFCore/CriteriaPropertiesEF/CriteriaPropertiesEF.Module/Controllers/CriteriaController.cs) 
-* [FilteringCriterion.cs](CS/EFCore/CriteriaPropertiesEF/CriteriaPropertiesEF.Module/BusinessObjects/FilteringCriterion.cs) 
-<!-- default file list end -->
-# How to use Criteria Property Editors
+# XAF - How to use Criteria Property Editors
 
+This example designs and saves filtering criteria at runtime in an XAF application.
```
Comment (albertov05) [inline Readme.md]:
```suggestion
This example designs and saves filter criteria at runtime in an XAF application.
```

File: Readme.md [hunk 2]
```diff
@@ -1,24 +1,26 @@
...
-<br/>
+## Files to Review
 
+- [CriteriaController.cs](CS/EFCore/CriteriaPropertiesEF/CriteriaPropertiesEF.Module/Controllers/CriteriaController.cs) 
+- [FilteringCriterion.cs](CS/EFCore/CriteriaPropertiesEF/CriteriaPropertiesEF.Module/BusinessObjects/FilteringCriterion.cs) 
+
+## Documentation
+
+- [Criteria Properties](https://docs.devexpress.com/eXpressAppFramework/113564/business-model-design-orm/data-types-supported-by-built-in-editors/criteria-properties)
+- [How to: Enable Database Storage for User-Defined Criteria that Filter a List View](https://docs.devexpress.com/eXpressAppFramework/113143/ui-construction/view-items-and-property-editors/property-editors/use-criteria-property-editors)
```
Comment (albertov05) [inline Readme.md:24]: Can you update line 28 below as follows?

`(You will be redirected to DevExpress.com to submit your response.)`

Comment (khamlyuk) [reply Readme.md:24]: Hi Albert! This section is generated automatically and the texts were prepared by Ray and Bogdan. I'll pass your feedback to them.

Comment (albertov05) [reply Readme.md:24]: Okay, got it. In that case let's leave it as is. Thanks!

**Pull request merged**